addons/explosives/stringtable.xml

```diff
@@ -505,6 +505,7 @@
             <Italian>Raccogli</Italian>
             <Portuguese>Pegar</Portuguese>
         </Key>
+<<<<<<< HEAD
```

Comment:
`<<<<< HEAD` - from merge conflict

for stringtables delete lines that aren't translated